src/components/modals/BtcSendModal.vue

```diff
@@ -190,6 +190,8 @@ export default defineComponent({
         } = useBtcLabelsStore();
         const { state: network$, isFetchingTxHistory } = useBtcNetworkStore();
 
+        const durationConfirmation = ref(3 * 60 * 60); // Fee selector has 2-4h option by default. Setting it as 3h.
```

Comment:
This is actually what one of the comments of Julian in the Figma was about. Since the designs were made, the way of displaying the duration changed from a single number to a range (2 _to_ 4 hours). So this needs to be displayed the same way in the Keyguard than it is in the Wallet. Meaning the `duration` parameter must be a range. Maybe `[from: number, to: number]`?

Reply:
I thinks this is a **great** solution! You basically pass the number of blocks that the user selects as the `delay`, which means the Keyguard does the same calculation on the same basis (number of blocks delay) as the wallet, which is the most consistent! It remains to be seen if we need to rename the variable to `blockDelay`.

For what it's worth, passing a string would be horrible, because it would not allow translations, nor any validation really.